test/integration/setup.js

```diff
@@ -0,0 +1,10 @@
+before(() => {
+  collectionName = externalConfig.collectionName;
```

Comment:
These are set globally? I think it's better to have one config object and keep things in there, if we must use global variables. That way we are not polluting the global scope with N variables, just the one.